json/src/lib.rs

```diff
@@ -123,6 +123,45 @@ pub struct LoadWalletResult {
     pub warning: Option<String>,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct GetWalletInfoResult {
+    #[serde(rename = "walletname")]
+    pub wallet_name: String,
+    #[serde(rename = "walletversion")]
+    pub wallet_version: u32,
+    #[serde(with = "bitcoin::util::amount::serde::as_btc")]
+    pub balance: Amount,
+    #[serde(with = "bitcoin::util::amount::serde::as_btc")]
+    pub unconfirmed_balance: Amount,
+    #[serde(with = "bitcoin::util::amount::serde::as_btc")]
+    pub immature_balance: Amount,
+    #[serde(rename = "txcount")]
+    pub tx_count: usize,
+    #[serde(rename = "keypoololdest")]
+    pub keypool_oldest: usize,
+    #[serde(rename = "keypoolsize")]
+    pub keypool_size: usize,
+    #[serde(rename = "keypoolsize_hd_internal")]
+    pub keypool_size_hd_internal: usize,
+    pub unlocked_until: Option<usize>,
```

Comment:
I double checked but all the timestamps seem to be either `usize` or `u64`. Changed to `u64` as it might be better and I don't mind changing to `SystemTime` but I feel it might be inconsistent.